dataform/examples/dataform_datavault/README.md

```diff
@@ -56,33 +56,16 @@ Meter Readings Staging table:
 2. [Google cloud project](https://developers.google.com/workspace/guides/create-project) should be available
 3. BigQuery api is enabled
 4. Access to create service account keys
-5. [Dataform lib ](https://docs.dataform.co/dataform-cli) lib using NPM
+5. [Dataform lib ](https://docs.dataform.co/dataform-cli) using NPM
 
 
 ### Steps to follow:
 All the steps below can be used via terminal or Use the CloudShell editor.
 
 In order to test and run the dataform, please run the following steps:
 
-1. Initialize a Dataform project and then create a new Dataform google cloud project ([*optional if you already have one*])
-    ```shell
-    dataform init bigquery dataform-demo --default-database $(gcloud config get-value project) --include-schedules
-    ```
+1. Set up [authentication](https://github.com/GoogleCloudPlatform/bigquery-utils/blob/master/dataform/examples/dataform_udf_unit_test/tutorial.md#2-generate-the-dataform-credentials-file-by-running-the-following) to BigQuery 
```

Comment:
Please copy the actual content here instead of linking to another page to not confuse the user

Reply:
Done.